library/Icingadb/Common/Auth.php

```diff
@@ -85,6 +85,19 @@ public function isGrantedOnType($permission, $type, Filter\Rule $filter, $cache
         return ObjectAuthorization::grantsOnType($permission, $type, $filter, $cache);
     }
 
+    /**
+     * Check whether the filter matches the given object
+     *
+     * @param string $restriction
+     * @param Model $object
+     *
+     * @return bool
+     */
+    public function isMatchedOn($restriction, Model $object)
```

Comment:
`isMatchedBy` sounds better, doesn't it?

Reply:
Or just simply `matches`?

Reply:
I like the attempt to maintain the `is` naming scheme.

Reply:
Well then, I'd say `matchesTo` would be better.

Reply:
Oh you're phrasing the method name based on the order of the parameters? With that in mind, yes, `isMatchedBy` doesn't feel correct. But `isMatchedTo` does neither, so let's keep `isMatchedOn`. :laughing: